packages/components/pages/f-checkout/src/components/Notes.vue

```diff
@@ -0,0 +1,113 @@
+            :title="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.text`) }}</span>
+            <form-field
+                :placeholder="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.placeholder`)"
+                :value="noteValue"
+                v-bind="inputStyles"
+                :name="`${noteTypeCourierOrOrder}-note`"
+                @input="updateUserNotes({ note: $event, type: noteTypeCourierOrOrder })" />
+        </accordion>
+        <accordion
+            v-if="shouldShowKitchenNotes"
+            id="kitchen"
+            :title="$t(`userNote.kitchen.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.kitchen.${serviceType}.text`) }}</span>
+            <form-field
+                :placeholder="$t(`userNote.kitchen.${serviceType}.placeholder`)"
+                :value="kitchenNoteValue"
+                v-bind="inputStyles"
+                name="kitchen-note"
+                @input="updateUserNotes({ note: $event, type: 'kitchen' })" />
+        </accordion>
+    </div>
+    <form-field
+        v-else-if="!isSplitNotesEnabled"
+        :label-text="$t(`userNote.order.delivery.title`)"
+        :placeholder="$t(`userNote.order.${serviceType}.placeholder`)"
+        :value="noteValue"
+        v-bind="inputStyles"
+        :name="'order-note'"
+        @input="updateUserNotes({ note: $event, type: 'order' })" />
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex';
+import FormField from '@justeat/f-form-field';
+import Accordion from './Accordion.vue';
+import '@justeat/f-form-field/dist/f-form-field.css';
+import {
+    CHECKOUT_NOTE_TYPE_ORDER,
+    CHECKOUT_NOTE_TYPE_COURIER,
+    VUEX_CHECKOUT_MODULE
+} from '../constants';
+import loggerMixin from '../mixins/logger.mixin';
+
+export default {
+    components: {
+        Accordion,
+        FormField
+    },
+
+    mixins: [
+        loggerMixin
+    ],
+
+    props: {
+        isSplitNotesEnabled: {
+            type: Boolean,
+            default: false
+        }
+    },
+
+    computed: {
+        ...mapState(VUEX_CHECKOUT_MODULE, [
+            'notesConfiguration',
+            'serviceType',
+            'notes'
+        ]),
+
+        shouldShowKitchenNotes () {
+            return this.notesConfiguration[this.serviceType]?.kitchenNoteAccepted;
+        },
+
+        noteTypeCourierOrOrder () {
+            return this.notesConfiguration[this.serviceType]?.courierNoteAccepted ? CHECKOUT_NOTE_TYPE_COURIER : CHECKOUT_NOTE_TYPE_ORDER;
+        },
+
+        noteValue () {
+            return this.noteTypeCourierOrOrder === CHECKOUT_NOTE_TYPE_COURIER ? this.notes.courier?.note : this.notes.order?.note;
+        },
+
+        kitchenNoteValue () {
+            return this.notes.kitchen?.note || '';
+        },
+
+        inputStyles () {
```

Comment:
Assuming this is the last remaining computed property, it could be moved into `data` seeing as it doesn't need to be recomputed